pyqldb/execution/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Why is this file added here?

Reply:
This file was updated. It was missing the license header

See: [file in master branch](https://github.com/awslabs/amazon-qldb-driver-python/blob/master/pyqldb/execution/__init__.py)